### PR TITLE
Adding List type conversion

### PR DESCRIPTION
**Pull request by myrnazhou**
Adding support to convert `List` from `.Net` to `Native`

### CODE REVIEW

File: CefSharp.Core/Internals/TypeConversion.h [hunk 1]
```diff
@@ -149,6 +149,18 @@ namespace CefSharp
                 {
                     cefValue->SetDouble(Convert::ToDouble(value));
                 }
+				else if (type == List<Object^>::typeid)
```
Comment (amaitland) [inline CefSharp.Core/Internals/TypeConversion.h]: Formatting is out, spaces not tabs.

Comment (myrnazhou) [inline CefSharp.Core/Internals/TypeConversion.h]: Sorry, I thought I had the space settings for all languages. I've pushed the changes.

Comment (amaitland) [inline CefSharp.Core/Internals/TypeConversion.h]: No problem. If you check below you'll see the formatting is still way out.

**Comment by amaitland** [comment]
Looks fine now :+1: Will merge this into `cefsharp/51` at some point soon.

**Comment by myrnazhou** [comment]
Cool, thanks.

**Comment by tuxan** [comment]
@myrnazhou  This mean we could use List inside of bound classes?

```c#
public class BoundObject
{
   private List<SubBoundObject> items { get; set; }
   public List<SubBoundObject> SomeItems { get; set; }
   ...
   public List<SubBoundObject> GetItems
   {
      return items;
   }
}
```

**Pull request merged**